Auth0/SafariWebAuth.swift

```diff
@@ -196,13 +212,21 @@ class SafariWebAuth: WebAuth {
     }
 
     func handler(_ redirectURL: URL) -> OAuth2Grant {
-        if self.responseType.contains([.code]) {
-            var authentication = Auth0Authentication(clientId: self.clientId, url: self.url, telemetry: self.telemetry)
+        var authentication = Auth0Authentication(clientId: self.clientId, url: self.url, telemetry: self.telemetry)
+        if self.responseType.contains([.code]) { // both Hybrid and Code flow
             authentication.logger = self.logger
-            return PKCE(authentication: authentication, redirectURL: redirectURL, reponseType: self.responseType, nonce: self.nonce)
-        } else {
-            return ImplicitGrant(responseType: self.responseType, nonce: self.nonce)
+            return PKCE(authentication: authentication,
+                        redirectURL: redirectURL,
+                        responseType: self.responseType,
+                        leeway: self.leeway,
+                        maxAge: self.maxAge,
+                        nonce: self.nonce)
         }
+        return ImplicitGrant(authentication: authentication,
+                             responseType: self.responseType,
+                             leeway: self.leeway,
+                             nonce: self.nonce,
+                             maxAge: self.maxAge)
```

Comment:
I guess these are named params and that's why you can call them in any order, right? Would be good anyway to keep them on the same calling order everywhere. e.g. `leeway`, `nonce`, `maxAge`. I saw something similar on the `OAuth2Grant` constructor and later on that same file when you instantiate `IDTokenValidatorContext`. 
Not a blocker, but a nice to have!

Reply:
These are named params, but they can't be called in any order. Only the ones with default values can be omitted. I made the order consistent in all cases in 721afba4d18a9453de055348a5e580602d885fb3. Thanks for raising it.